stable/database/templates/secret.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
   name: {{ template "database.secretName" . }}
 stringData:
   "database-name": {{ .Values.database.name | quote }}
+  "database-archive-type": {{ .Values.database.archiveType | quote }}
```

Comment:
Kubernetes secrets are usually used to store sensitive data which the archive type is not. I would suggest directly rendering the env variable rather than using references.

Reply:
+1

We should directly render the value in the same way we do for the variables in lines [559 to 565](https://github.com/nuodb/nuodb-helm-charts/blob/de236dc758025ad64ac6351eae2af5e7e92742f2/stable/database/templates/statefulset.yaml#L559).

Reply:
Done.